Dockerfile

```
@@ -0,0 +1,31 @@
+#
+## build stage
+FROM golang:1.20 as builder
```

You can also use the FATMAP Go image if you want

```suggestion
FROM 793842381218.dkr.ecr.eu-west-1.amazonaws.com/fatmap-golang:v1.20.0 as builder
```

---

Ignore this if it will be open-source

---

It is supposed to be open-source so I will stick to the vanilla go image :)